features/step_definitions/snippet_steps.rb

```diff
@@ -0,0 +1,23 @@
+Given(/^I am authenticated$/) do
+  user = User.new email: "landry.soules@gmail.com", password: "1Password", username: "landry"
+  signup user
+  authenticate user
+  page.assert_text "Signed in successfully."
+end
+
+When(/^I fill in the snippet creation form$/) do
+  #TODO: check why cucumber doesn't recognize routing helpers
+  #visit new_snipet
```

Comment:
Yeah, both of those are URL helpers provided by Rails (http://guides.rubyonrails.org/routing.html#path-and-url-helpers), difference is in presence of host ("/snipets/new" vs "http://localhost:3000/"/snipets/new")